src/Fields/Fields.jl

```diff
@@ -332,4 +332,20 @@ function level(field::FaceExtrudedFiniteDifferenceField, v::PlusHalf)
     Field(data, hspace)
 end
 
+"""
+    set!(field::Field, f::Field)
+    set!(field::Field, f::Function)
+
+Assign field `field` to values in
+`f`, which may be a field or a
+function.
```

Comment:
@valeriabarra, the inputs / outputs are explained here-- I'm following the julia convention that the first argument is mutated.

Reply:
Yes, thanks, I had seen that. I was just wondering if it was clearer to mention in the docstring that indeed the first argument was going to be the output, but it's ok if you think the convention is clear enough and commonly known

Reply:
Sure, can add a suggestion?

Reply:
If the first argument is the input/output, how does something like this sound?
 
```suggestion
Assign values in `f`, 
which may be a field or a
function, to field `field`.
```

Reply:
Or did I get what you wanted to say backward?

Reply:
This looks backwards to me, but maybe I've re-read this too many times. @simonbyrne, what do you think?

Reply:
Then I must have misunderstood what this does. I thought we said that the first argument (`field`) is both an input and the output (i.e., mutated by the function). Hence I thought the assignment was of the values of `f` to `field`. As in:
```
field <-- f
```

Reply:
I guess I was interpreting
```
Assign values in `f`
```
to mean that `f` is mutating, and that's not the case. I'm fine with this doc string if people find it more clear.

Reply:
Oh, how about 

```suggestion
Assign values of `f`, 
which may be a field or a
function, to field `field`.
```
?